video/out/w32_common.c

```diff
@@ -1202,6 +1201,17 @@ static void update_backdrop(const struct vo_w32_state *w32)
                           &backdropType, sizeof(backdropType));
 }
 
+static void set_ime_conversion_mode(const struct vo_w32_state *w32, DWORD mode) {
+    if (w32->parent)
+        return;
+
+    HIMC imc = ImmGetContext(w32->window);
+    DWORD conversion_mode, sentence_mode;
+    ImmGetConversionStatus(imc, &conversion_mode, &sentence_mode);
```

Comment:
`conversion_mode` is not used and is optional, you can pass NULL here.

Also check if get succeeded, to avoid calling Set next with incorrect `sentence_mode`